src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirModel.cs

```diff
@@ -388,11 +397,11 @@ COMMIT TRANSACTION
             }
         }
 
-        private void ThrowIfNotStarted()
+        private void ThrowIfNotInitialized()
         {
-            if (!_started)
+            if (_highestInitializedVersion < _schemaInformation.Current)
```

Comment:
>_highestInitializedVersion < _schemaInformation.Current [](start = 16, length = 55)

Can be wrong, but if I correctly read code, in case if nothing been called/initialize, `_highestInitializedVersion` would be 0 and `_schemaInformation.Current` can be `null`, `0<null` is false, and nothing would be thrown here.

Reply:
You're right, thanks for catching this 😊. Added a null check in 4ef0c98.